components/02-elements/price/Price.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <div class="price">
+    <ins
+      v-if="specialPrice"
+      :aria-label="setAriaLabel(specialPrice)"
+      class="price--special"
+    >
+      {{ specialPrice }}
+    </ins>
+    <del
+      v-if="oldPrice"
+      :aria-label="setAriaLabel(oldPrice)"
+      class="price--old"
+    >
+      {{ oldPrice }}
+    </del>
+    <span
+      v-if="price"
+      :aria-label="setAriaLabel(price)"
+    >
+      {{ price }}
+    </span>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    price: {
+      type: String,
+      default: null
+    },
+    specialPrice: {
+      type: String,
+      default: null
+    },
+    oldPrice: {
+      type: String,
+      default: null
+    },
+    ariaLabelPrice: {
+      type: String,
+      default: 'Price:'
+    },
+    ariaLabelSpecial: {
+      type: String,
+      default: 'Special price:'
+    },
+    ariaLabelOld: {
+      type: String,
+      default: 'Old price:'
+    }
+  },
+  methods: {
+    setAriaLabel(type) {
+      if (type === this.specialPrice) {
+        return `${this.ariaLabelSpecial} ${this.specialPrice}`
+      }
+      if (type === this.oldPrice) {
+        return `${this.ariaLabelOld} ${this.oldPrice}`
+      }
+      return `${this.ariaLabelPrice} ${this.price}`
+    }
+  }
+}
+</script>
+
+<style lang="scss">
```

Comment:
2 spaces indentation

Reply:
We have automatic formatting via eslint enabled, so this is how it's defined in `plugin:vue/recommended`.

Reply:
maybe it doesn't work for some reason b/c in some vue components I can see different formatting

Reply:
ok, eslisnt doesn't touch styles section